core/src/main/java/com/google/cloud/sql/core/CloudSqlInstance.java

```diff
@@ -282,6 +245,60 @@ private static Certificate createCertificate(String cert) throws CertificateExce
     return CertificateFactory.getInstance("X.509").generateCertificate(certStream);
   }
 
+  static void checkDatabaseCompatibility(ConnectSettings instanceMetadata, boolean iamAuth,
```

Comment:
Can we move this organizational cleanup into a separate PR?

Reply:
I use intelliJ to format my code and it's honestly more work to manually undo the formatting fixes we don't want. Can we just leave it for now?

Reply:
Actually I could probably make another PR with just the formatting and rebase this one when that's merged.